gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/sampling/statistics/AverageClusteringCoefficientTest.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.operators.sampling.statistics;
+
+import org.apache.commons.math3.util.Precision;
+import org.gradoop.common.model.impl.pojo.GraphHead;
+import org.gradoop.flink.model.GradoopFlinkTestBase;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.util.FlinkAsciiGraphLoader;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test-class for {@link AverageClusteringCoefficient}
+ */
+public class AverageClusteringCoefficientTest extends GradoopFlinkTestBase {
+
+  /**
+   * GraphLoader used for test-cases
+   */
+  protected FlinkAsciiGraphLoader loader;
+
+  /**
+   * Constructor
+   */
+  public AverageClusteringCoefficientTest() {
```

Comment:
There is a `@Before` annotation to define a setup function where something like this can be done. Using a constructor in a test is not the common way.

Reply:
Changed it